html/infrastructure/urls/resolving-urls/query-encoding/navigation.sub.html

```diff
@@ -81,10 +81,19 @@
   }, 'follow hyperlink <'+tag+' href>');
 }
 
-'a, area, link'.split(', ').forEach(function(str) {
+'a, area'.split(', ').forEach(function(str) {
   test_follow_link(str);
 });
 
+async_test(function() {
```

Comment:
I'm not seeing this test fail in Firefox when I try it locally; are you?

Reply:
Indeed, https://wptpr.live/27178/html/infrastructure/urls/resolving-urls/query-encoding/navigation.sub.html shows it still passing.

Reply:
It was failing in firefox for me, but it was passing in safari which it shouldnt be, so I just changed it a bit - how about now?

Reply:
Looks good; failing in Firefox now! https://wptpr.live/27178/html/infrastructure/urls/resolving-urls/query-encoding/navigation.sub.html